module1.js

```diff
@@ -80,7 +83,18 @@ function question5(number1, number2) {
 */
 
 function question6(array) {
-  // TODO:
+  var sum = array.length;
+  return (
+    "The array contains " +
+    sum +
+    " animals. These are: " +
+    array[0] +
```

Comment:
So this passes the test, only because we pass an array that only has 3 elements, in the test. Sadly, the function will no longer work as expected, if we try it with an array that has only 2 animals in it, or 10.
Can you think of a function that lets you list all the array elements?